### PR TITLE
Feature/basic syntax

### PR DESCRIPTION
**Pull request by justy989**
-input with loading file, and searching
-window split characters
-basic syntax highlighting

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,34 +1,6 @@
...
-*.out
-*.app
-*.i*86
-*.x86_64
-*.hex
 
-# Debug files
-*.dSYM/
-*.su
+# WHAT IS THIS AND WHY DOES IT EXIST?
```
Comment (robber-m) [inline .gitignore:5]: lol

File: ce.c [hunk 2]
```diff
@@ -995,6 +1000,35 @@ bool ce_draw_buffer(const Buffer* buffer, const Point* term_top_left, const Poin
...
+          "NULL",
+          "case",
+          "typedef",
+          "default",
+          "struct",
+          "static",
+          "inline",
+          "volatile",
+          "extern",
+          "__thread",
```
Comment (ryabouza) [inline ce.c:1031]: Switch

Comment (ryabouza) [reply ce.c:1031]: Do

Comment (ryabouza) [reply ce.c:1031]: Union

**Review by robber-m — APPROVED** [review]
looks good

**Review by ryabouza — APPROVED** [review]

**Review by robber-m — APPROVED** [review]

**Pull request merged**